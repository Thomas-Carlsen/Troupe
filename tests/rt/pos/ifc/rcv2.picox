let val secret = 42 raisedTo {secret}
    val _ = send (self(), "public message")
    val _ = if secret > 0
            then send (self(), "secret message")
            else send (self(), "other secret message")
in rcvp ({secret},  [hn x => printWithLabels x]) end
