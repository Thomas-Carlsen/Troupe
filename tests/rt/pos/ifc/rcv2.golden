2019-03-08T06:10:03.480Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
"secret message"@{secret}%{secret}
>>> Main thread finished with value: ()@{secret}%{secret}
